phpunit/functional/Glpi/Inventory/InventoryTest.php

```diff
@@ -8454,4 +8454,285 @@ public function testPartialUser()
         $this->assertTrue($computer->getFromDBByCrit(['name' => 'pc_with_user']));
         $this->assertSame($newuser, $computer->fields['contact']);
     }
+
+    public function testRuleRefuseImportComputerVirtualMachines()
+    {
+        //change config to import vms as computers
+        $this->login();
+        $conf = new \Glpi\Inventory\Conf();
+        $this->assertTrue($conf->saveConf(['vm_as_computer' => 1]));
+        $this->logout();
+
+        $nb_vm = countElementsInTable(\VirtualMachineType::getTable());
+        $docker_vm = $this->createItems(
+            \VirtualMachineType::class,
+            [
+                ['name' => 'docker'],
+            ]
+        );
+
+        $this->assertGreaterThan($nb_vm, countElementsInTable(\VirtualMachineType::getTable()));
+
+        //IMPORT rule to refuse "db" virtual machine
```

Comment:
There is no "db" vm.

Also, as far as I've understood, this rule is to prevent import of "docker" VMs, but all types from example are docker and are imported. 
Did I miss something?

(I've jsut checked this test for now).